src/app/search/components/patient_search/patient_search.html

```diff
@@ -7,12 +7,21 @@ <h2 class="panel-title">Patient Query</h2>
       </div>
       <div class="panel-body slide" ng-if="!vm.hidePanel">
         <div class="form-group">
-          <label class="col-sm-12 control-label">Name<span ng-if="vm.query.patientNames && vm.query.patientNames.length !== 1">s</span> <span class="text-danger">*</span></label>
-          <div class="col-sm-12">
-            <ai-names names="vm.query.patientNames"
-                      active-form="vm.queryForm"
-                      show-form-errors="vm.showFormErrors"
-                      submit-form="vm.searchForPatient"></ai-names>
+        	<label class="col-sm-12 control-label">Name<span class="text-danger">*</span></label>
```

Comment:
Needs a space after "Name" before <span..., otherwise the `*` will abut the title, which it doesn't anywhere else